src/serverlist.cpp

```diff
@@ -184,12 +184,13 @@ CServerListManager::CServerListManager ( const quint16  iNPortNum,
     /*
      * Init server list entry (server info for this server) with defaults.
      *
-     * The client will use the built in or custom address when retrieving the server list from a directory.
```

Comment:
Removed on purpose?

Reply:
Yes.  This block comment doesn't describe _client_ behaviour.  It's about the _server_ (whether a _directory_ or not).